src/components/buttons/Button.vue

```diff
@@ -1,49 +1,73 @@
 <script lang="ts" setup>
 import { type ContextColorsType } from '@/consts/colors';
 
-withDefaults(
+const props = withDefaults(
   defineProps<{
     disabled?: boolean;
     loading?: boolean;
     outlined?: boolean;
     color?: ContextColorsType;
-    tile?: boolean;
-    elevated?: boolean;
+    rounded?: boolean;
```

Comment:
do you think we can introduce variants here too?

Reply:
yeah I think so. 
The variant can be either `default` | `outlined` | `text`
the other prop can be used simultaneously

Reply:
Can fab be mixed with other variants?

Reply:
fab can be used with icon, but not with outlined | text I think. Okay I can put it in variant too.